### DISCUSSION TITLE
Suggestion for a simple and easy automatic dark theme by enabling preset style selector from the public frontend

### DISCUSSION THREAD

**Discussion by xunga87** [gantry/gantry5#3280] [Ideas]
Hello, I do not know if this is the right place to make this suggestion.

I have been using the Helium template for a long time, first in Joomla and now in Worpress, and I have seen over time several open issues requesting the integration of the dark mode in the Gantry themes.

I usually use Helium in the preset 1 style which would be a light theme, but when I wanted to create a website in dark mode I have used the preset 6 style and it works quite well.

It would be ideal to have the option to put in the template a toggle button so that web visitors could choose the css style of their preference as we do from the administration panel.

This way I could section my default base style from the administration and allow the user to choose the preset 6 style from the public frontend and have a sort of dark mode without having to do any extra development work in Gantry 5 since we would use the css styles that are already configured by default.

If this could be implemented it would be ideal.

In any case, thank you very much for your work and attention to the community.

Gantry, in particular Helium, seems to me the best template.

**Comment by 4digitalworld** [comment]
adding `Dark Mode` is not that difficult,  it can be done using `prefers-color-scheme`, I hope the @Gantry developers will add a button to switch `Light`/`Dark`/`Auto` 
[https://web.dev/articles/prefers-color-scheme#the_prefers-color-scheme_media_query](https://web.dev/articles/prefers-color-scheme#the_prefers-color-scheme_media_query)
